.github/workflows/node-publish-aws-lambda-otel-extension.yml

```diff
@@ -22,43 +22,43 @@ jobs:
         uses: actions/cache@v2
         with:
           path: ~/.npm
-          key: npm-v16-${{ runner.os }}-refs/heads/main-${{ hashFiles('**package*.json') }}
+          key: npm-v16-${{ runner.os }}-refs/heads/main-${{ hashFiles('node/**package*.json') }}
           restore-keys: npm-v16-${{ runner.os }}-refs/heads/main-
 
       - name: Retrieve node_modules from cache
         id: cacheNodeModules
         uses: actions/cache@v2
         with:
-          path: node_modules
-          key: node-modules-v16-${{ runner.os }}-refs/heads/main-${{ hashFiles('package.json') }}
+          path: node/node_modules
+          key: node-modules-v16-${{ runner.os }}-refs/heads/main-${{ hashFiles('node/package.json') }}
           restore-keys: node-modules-v16-${{ runner.os }}-refs/heads/main-
       - name: Retrieve packages/aws-lambda-otel-extension/node_modules from cache
         id: cacheAwsLambdaOtelExtensionNodeModules
         uses: actions/cache@v2
         with:
-          path: packages/aws-lambda-otel-extension/node_modules
-          key: packages/aws-lambda-otel-extension/node-modules-v16-${{ runner.os }}-refs/heads/main-${{ hashFiles('packages/aws-lambda-otel-extension/package.json') }}
+          path: node/packages/aws-lambda-otel-extension/node_modules
+          key: packages/aws-lambda-otel-extension/node-modules-v16-${{ runner.os }}-refs/heads/main-${{ hashFiles('node/packages/aws-lambda-otel-extension/package.json') }}
           restore-keys: packages/aws-lambda-otel-extension/node-modules-v16-${{ runner.os }}-refs/heads/main-
       - name: Retrieve packages/aws-lambda-otel-extension/opt/otel-extension/node_modules from cache
```

Comment:
And similar comment about the `name` field in this file. Might be worth updating all the names as well 🤷